org.mixedrealitytoolkit.uxcomponents/Slider/CanvasSlider.prefab

```diff
@@ -102,6 +102,7 @@ MonoBehaviour:
   _Iridescent_Map_: {fileID: 0}
   _Frequency_: 1.89
   _Vertical_Offset_: 0
+  _Orthographic_Distance_: 400
```

Comment:
are all of these prefab changes desired?

Reply:
these MIGHT have happened because I was using Unity 2021.3.30f1. There are a number of packages that do not load on 2021.3.16f1 that are referenced in the sample project manifest.

Reply:
2021.3.21f1 (matching version) also added these entries. it appears that Unity changed serialization. Unfortunately, it is not clear what version caused these to be added (and the project does not load with earlier versions due to package incompatibilities)

Reply:
ah okay, makes sense. usually for prefab changes I will just manually unstage anything extraneous that Unity does / manually stage the changes I want

Reply:
(also hopefully the project will load with earlier versions of 2021 after the recent changes to avoid compile errors)

Reply:
the project is not loadable in previous versions due to an updated Unity package (not sure which one) that is not compatible with 16f1, for example.

Reply:
I was able to run the samples in 16f1 after this change https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/484, are there other errors in 16f1?

Reply:
i'l double check. there _had_ been a package issue in the past.

Reply:
@marlenaklein-msft, you are correct. the template project opens successfully in 16f1 :smile:

Reply:
resolving: this change is a result of reserializing with 2021.3. Any future change will generate these changes. Let's keep them now, so we do not have this same discussion at that time.  😄